### PR TITLE
Fix release issues

### PR DESCRIPTION
**Pull request by dreampulse**

### CODE REVIEW

File: src/app/container/upload-page.js [hunk 1]
```diff
@@ -83,11 +84,7 @@ const UploadPage = ({
             Large quantities, recurring orders or special requirements?
           </RichText>
           <div className="u-align-center ">
-            <Button
-              minor
-              label="Contact Us"
-              onClick={() => global.window.open('https://craftcloud.all3dp.com/contact', '_blank')}
-            />
+            <Button minor label="Contact Us" onClick={() => openIntercom()} />
```
Comment (dreampulse) [inline src/app/container/upload-page.js]: This is just a hotfix. This should be a link with a `mailto:` fallback.
There are other places which needs to be changed as well

File: src/app/service/intercom.ts [hunk 2]
```diff
@@ -6,21 +6,19 @@ const wellKnownIntercomMethodNames = ['boot', 'shutdown']
...
-
   return wellKnownIntercomMethodNames.every(methodName => intercomImpl.includes(methodName))
 }
 
-function exec(cmd: string): void {
-  if (isActualIntercomImpl() === false) {
-    throw new TypeError('Intercom function validation failed')
+const intercom = (cmd: string): void => {
+  if (window.Intercom) {
+    window.Intercom(cmd)
```
Comment (dreampulse) [inline src/app/service/intercom.ts]: The `throw` here causes the application to crash if there is a adblocker...

File: src/app/container/modal/address-form.js [hunk 3]
```diff
@@ -186,115 +187,114 @@ const AddressFormModal = ({
...
-
-          <FormRow>
-            <Field
-              name="useDifferentBillingAddress"
-              component={renderFormikField(LabeledCheckbox)}
-              label="Use different billing address"
-            />
-          </FormRow>
-        </div>
+        <div id="shipping-address" />
```
Comment (dreampulse) [inline src/app/container/modal/address-form.js]: This div messed up with the spacings. We may need to rethink the spacing

File: src/app/component/button/button.js [hunk 4]
```diff
@@ -21,19 +22,27 @@ const Button = ({
   block = false,
   selected = false,
   iconOnly = false,
-  onClick = () => {}
+  onClick = noop,
+  asLink = false,
```
Comment (webholics) [inline src/app/component/button/button.js]: können wir anhand des href erkennen, daher is das prop unnötig

File: src/app/component/button/button.story.js [hunk 5]
```diff
@@ -28,3 +28,68 @@ storiesOf('Button', module)
   .add('iconOnly & disabled', () => (
     <Button icon={placeholderIcon} disabled iconOnly onClick={action('onClick')} />
   ))
+
+  .add('default & asLink', () => (
```
Comment (webholics) [inline src/app/component/button/button.story.js]: bitte nicht alle kombinationen hier aufführen, das is viel zu viel und zu unübersichtlich

**Pull request merged**